test/integration/scheduler/scheduler_test.go

```diff
@@ -211,16 +285,7 @@ func TestSchedulerCreationInLegacyMode(t *testing.T) {
 	ss.PolicyConfigMapName = "non-existent-configmap"
 	ss.UseLegacyPolicyConfig = true
 
-	sched, err := app.CreateScheduler(ss, clientSet,
-		informerFactory.Core().V1().Nodes(),
-		informerFactory.Core().V1().PersistentVolumes(),
-		informerFactory.Core().V1().PersistentVolumeClaims(),
-		informerFactory.Core().V1().ReplicationControllers(),
-		informerFactory.Extensions().V1beta1().ReplicaSets(),
-		informerFactory.Apps().V1beta1().StatefulSets(),
-		informerFactory.Core().V1().Services(),
-		eventBroadcaster.NewRecorder(api.Scheme, clientv1.EventSource{Component: v1.DefaultSchedulerName}),
-	)
+	sched, err := app.CreateScheduler(ss, clientSet, informerFactory,	eventBroadcaster.NewRecorder(api.Scheme, clientv1.EventSource{Component: v1.DefaultSchedulerName}))
```

Comment:
Oh, I guess you are referring to what is shown here before `eventBroadcaster`. It is not in the code. Maybe it is a github issue in showing diffs.